src/charactersheet/viewmodels/dm/encounter_sections/monster_section/form.html

```diff
@@ -122,6 +122,34 @@
     </div>
     <div class="form-group col-xs-12 col-lg-6">
       <label>Hit Points<i class="required"></i></label>
+      <input
+        type="text"
+        class="form-control"
+        name="maxHitPoints"
+        placeholder="367 (21d20 + 147)"
+        data-bind="
+          value: maxHitPoints,
+          event: { blur: $component.reviewInput, invalid: $component.invalidate },
+          attr: { ...$component.validation.maxHitPoints },
+        "
+      />
+    </div>
+    <div class="form-group col-xs-12 col-lg-6">
+      <label>Damage<i class="required"></i></label>
+      <input
+        type="text"
+        class="form-control"
+        name="damage"
+        placeholder="367 (21d20 + 147)"
+        data-bind="
+          value: damage,
+          event: { blur: $component.reviewInput, invalid: $component.invalidate },
+          attr: { ...$component.validation.damage },
+        "
+      />
+    </div>
+    <div class="form-group col-xs-12 col-lg-6">
+      <label>Hit Dice<i class="required"></i></label>
```

Comment:
This is a backwards compatibility issue. If we add 'maxHitPoints' to manage hit points that require math (which I labeled 'Hit Points'), it makes sense to relabel the current hit points (which is formatted like `12 (2d8)`. I chose 'Hit Dice' as a reasonable description of this data.

Reply:
I've removed maxHitPoints and damage from monsters in the dm section so that more efforts could be put into that update/migration in a future release. This removes the ambiguity, and still allows companions to launch.